data/codes/550.json

```
@@ -510,6 +510,27 @@
           "description": "",
           "links": []
         },
+        {
+          "status": "5.1.8",
+          "response": "smtp;550 5.1.8 <sender@example.com>: Sender address rejected: Domain not found (in reply to RCPT TO command)",
+          "severity": 0,
+          "description": "The domain of the sender is not registered and does not exist.",
+          "links": []
+        },
+        {
+          "status": "5.1.8",
```

Looks like there's already a `5.1.8` status for this provider so we'll need to change this to `5.1.8-2`. The status is used for URL generation and for React keys, so we'll need to make sure that they are unique. I may need to add a note to the documentation about that.

```suggestion
          "status": "5.1.8-2",
```